etc/win-ci/cygwin-build-iconv.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
+# shellcheck disable=SC2155
 
-set -eo pipefail
+set -e
 
-Version=$1
+# Version=$1 # unused variable
```

Comment:
@HertzDevil Should the version be used for anything? If not, we could also consider removing the argument entirely.

Reply:
no (also remove the corresponding `"$Version"` argument in `etc/win-ci/build-iconv.ps1`)